lib/snyk/enrich_spdx.go

```diff
@@ -35,6 +36,47 @@ const (
 	snykVulnerabilityDB_URI = "https://security.snyk.io"
 )
 
+type spdxEnricher = func(*spdx_2_3.Package, packageurl.PackageURL)
```

Comment:
can we also do this?
```suggestion
type spdxEnricher = func(*spdx_2_3.Package, *packageurl.PackageURL)
```